modules/claims_api/app/controllers/concerns/claims_api/poa_verification.rb

```diff
@@ -105,21 +104,15 @@ def valid_poa_code_for_current_user?(poa_code) # rubocop:disable Metrics/MethodL
       def verify_power_of_attorney!
         return if token.client_credentials_token?
 
-        logged_in_representative_user = @current_user
-        target_veteran_to_be_verified = target_veteran
-        verify_representative_and_veteran(logged_in_representative_user, target_veteran_to_be_verified)
+        target_veteran_to_verify = ::Veteran::User.new(target_veteran)
+        poa_code_to_verify = target_veteran_to_verify.power_of_attorney.try(:code)
+        valid_poa_code_for_user?(poa_code_to_verify, @current_user)
       rescue ::Common::Exceptions::UnprocessableEntity
         raise
       rescue
         raise ::Common::Exceptions::Unauthorized, detail: 'Cannot validate Power of Attorney'
       end
 
-      def verify_representative_and_veteran(logged_in_representative_user, target_veteran_to_be_verified)
-        verifying_bgs_service = BGS::PowerOfAttorneyVerifier.new(target_veteran_to_be_verified)
-        verifying_bgs_service.verify(logged_in_representative_user)
```

Comment:
Backlink for JIRA comment. The `BGS::PowerOfAttorneyVerifier` also has [dependencies on BGS](https://github.com/department-of-veterans-affairs/vets-api/blob/3dac695f63ef2aadadb5c702c18daf0c00e909c0/modules/veteran/app/models/veteran/user.rb#L29) to lookup POA codes, so there are cascading changes + test refactoring that might be beyond the scope of this work.

Reply:
On second thought, we only want to remove the duplicate validation logic. We can still rely on BGS for now to look up the POA code (to be removed later). Will refactor the (many) failing tests that relied on `BGS::PowerOfAttorneyVerifier`.